certificate-policy.md

```diff
@@ -1264,15 +1276,28 @@ Subscriber Certificates issued for delegated OCSP responders SHALL have a Validi
 
 ### 6.5.1 Specific computer security technical requirements
 
-The CA SHALL enforce multi-factor authentication for all accounts capable of directly causing certificate issuance.
-
-For all CAs operating under this policy, the computer security functions listed below are required. These functions may be provided by the operating system, or through a combination of operating system, software, and physical safeguards. The CA and its ancillary parts shall include the following functionality:  
+Administrator privileges to all Certificate System components SHALL only be granted to the Administrator trusted role. The CA SHALL enforce multi-factor authentication using a unique credential created by or assigned to the trusted role for all accounts capable of directly causing certificate issuance or authenticating to Certificate Systems. Multifactor authentication SHALL be implemented on all Certificate System components. If multifactor authentication is not supported or is not technically feasible to implement, the following username and password controls, where technically feasible, SHALL be implemented:
+
+- For accounts that are not publicly accessible (accessible only within a Secure or High Security Zone), require passwords have at leaast twelve (12) characters.
```

Comment:
Correct, Two factor may be difficult for air-gapped or offline systems. I think defining air-gapped and offline will be helpful as well. Air-gapped meaning no internet connection and physically separate from connected systems while Offline is a powerdowned air-gapped systems only turned on for specific certificate actions. One thing the CAB Forum Policy Working Group attempted to do was define clearly identify which requirements to which type of CA (differentiating between issuing and root CAs). Multifactor should be a requirement for online systems, but may not be a requirement for offline or air-gapped based on the commensurate physical and logical controls.